diff --git a/node_modules/hexo-theme-butterfly/layout/includes/head.pug b/node_modules/hexo-theme-butterfly/layout/includes/head.pug
index 0038afc..73093af 100644
--- a/node_modules/hexo-theme-butterfly/layout/includes/head.pug
+++ b/node_modules/hexo-theme-butterfly/layout/includes/head.pug
@@ -21,6 +21,8 @@ meta(name="author" content=pageAuthor)
 meta(name="copyright" content=pageCopyright)
 meta(name ="format-detection" content="telephone=no")
 meta(name="theme-color" content=themeColor)
+meta(name="baidu-site-verification" content="codeva-6t17LBl2wH")
+
 
 //- Open_Graph
 include ./head/Open_Graph.pug
diff --git a/node_modules/hexo-theme-butterfly/source/css/_layout/aside.styl b/node_modules/hexo-theme-butterfly/source/css/_layout/aside.styl
index f9c1bae..790070d 100644
--- a/node_modules/hexo-theme-butterfly/source/css/_layout/aside.styl
+++ b/node_modules/hexo-theme-butterfly/source/css/_layout/aside.styl
@@ -114,6 +114,7 @@
         overflow: hidden
         width: w = 4.2em
         height: w
+        border-radius: 4px
 
         :first-child
           @extend .imgHover
@@ -164,8 +165,7 @@
 
       &:hover
         padding: 3px 17px
-        background-color: var(--text-bg-hover)
-
+        color:#2563eb
       span
         @extend .limit-one-line
 
@@ -329,7 +329,7 @@
     object-fit: cover
 
     &:hover
-      transform: rotate(360deg)
+      transform:scale(1.2)
 
 .site-data
   display: table
diff --git a/node_modules/hexo-theme-butterfly/source/css/_layout/comments.styl b/node_modules/hexo-theme-butterfly/source/css/_layout/comments.styl
index f2feebd..82f4f94 100644
--- a/node_modules/hexo-theme-butterfly/source/css/_layout/comments.styl
+++ b/node_modules/hexo-theme-butterfly/source/css/_layout/comments.styl
@@ -61,7 +61,6 @@
   .comment-wrap
     > div
       animation: tabshow .5s
-
       &:nth-child(2)
         display: none
 
diff --git a/node_modules/hexo-theme-butterfly/source/css/_layout/head.styl b/node_modules/hexo-theme-butterfly/source/css/_layout/head.styl
index 97cb0ff..0fe9682 100644
--- a/node_modules/hexo-theme-butterfly/source/css/_layout/head.styl
+++ b/node_modules/hexo-theme-butterfly/source/css/_layout/head.styl
@@ -272,6 +272,7 @@
   &.show
     opacity: 1
   
+
   #blog-info
     flex: 1
     color: var(--light-grey)
@@ -343,7 +344,7 @@
           list-style: none
 
           &:hover
-            background: var(--text-bg-hover)
+            background: #e5e7eb
 
           &:first-child
             border-top-left-radius: 5px
diff --git a/node_modules/hexo-theme-butterfly/source/css/_layout/sidebar.styl b/node_modules/hexo-theme-butterfly/source/css/_layout/sidebar.styl
index d960fff..7df228f 100644
--- a/node_modules/hexo-theme-butterfly/source/css/_layout/sidebar.styl
+++ b/node_modules/hexo-theme-butterfly/source/css/_layout/sidebar.styl
@@ -45,7 +45,7 @@
         border-radius: 6px
 
         &:hover
-          background: var(--text-bg-hover)
+          background: #e5e7eb
 
         i:first-child
           width: 15%
diff --git a/node_modules/hexo-theme-butterfly/source/css/_page/flink.styl b/node_modules/hexo-theme-butterfly/source/css/_page/flink.styl
index 57dd50a..7fa32c2 100644
--- a/node_modules/hexo-theme-butterfly/source/css/_page/flink.styl
+++ b/node_modules/hexo-theme-butterfly/source/css/_page/flink.styl
@@ -17,6 +17,8 @@
         border-radius: 8px
         line-height: 17px
         -webkit-transform: translateZ(0)
+        background: #f3f4f6
+
 
         +maxWidth1024()
           width: calc(50% - 15px) !important
@@ -36,7 +38,6 @@
           bottom: 0
           left: 0
           z-index: -1
-          background: var(--text-bg-hover)
           content: ''
           transition: transform .3s ease-out
           transform: scale(0)
diff --git a/node_modules/hexo-theme-butterfly/source/css/_page/tags.styl b/node_modules/hexo-theme-butterfly/source/css/_page/tags.styl
index bce33ae..d887886 100644
--- a/node_modules/hexo-theme-butterfly/source/css/_page/tags.styl
+++ b/node_modules/hexo-theme-butterfly/source/css/_page/tags.styl
@@ -1,12 +1,17 @@
 .tag-cloud
+ 
   &-list
     a
       display: inline-block
       padding: 0 8px
       transition: all .3s
+      background: rgba(226, 232, 240, 1)
+      border-radius: 4px
+      margin: 0 5px 5px 0
 
       &:hover
         color: $text-hover !important
+        background: rgba(226, 232, 240, 0.8)
         transform: scale(1.1)
 
       +maxWidth768()
